examples/tabs/index.html

```diff
@@ -1,582 +1,402 @@
-
-</pre>
-          </td>
-
-          <td class='output'></td>
-        </tr>
-
-        <tr>
-          <td class='input'>
-            <pre>
-         #  Add three more in the middle
-         &lt;#tabs&gt; a ui:Tabs;
-           ui:predicate trip:attachmentList;
-           ui:orientation 3; ui:ordered true;
-           ui:subject &lt;#it&gt;.
-         &lt;#it&gt;
-           trip:attachmentList ( &lt;http://aclu.org/&gt;  &lt;http://bbc.co.uk/&gt;  &lt;http://cbc.ca/&gt;  &lt;http://dbp.de/&gt;  &lt;https://webfoundation.org/&gt; ) .
-
-</pre>
-          </td>
-
-          <td class='output'></td>
-        </tr>
-
-        <tr>
-          <td class='input'>
-            <pre>
-          #  Add one more on each end
-          &lt;#tabs&gt; a ui:Tabs;
-            ui:predicate trip:attachmentList;
-            ui:orientation 3; ui:ordered true;
-            ui:subject &lt;#it&gt;.
-          &lt;#it&gt;
-            trip:attachmentList ( &lt;http://acm.org/&gt;  &lt;http://aclu.org/&gt;
-            &lt;http://bbc.co.uk/&gt;  &lt;http://cbc.ca/&gt;  &lt;http://dbp.de/&gt;  &lt;https://webfoundation.org/&gt;
-            &lt;http://www.zenith.org/&gt; ) .
-
-</pre>
-          </td>
-
-          <td class='output'></td>
-        </tr>
-
-        <tr>
-          <td class='input'>
-            <pre>
-           #  Reverse the list
-           &lt;#tabs&gt; a ui:Tabs;
-             ui:predicate trip:attachmentList;
-             ui:orientation 3; ui:ordered true;
-             ui:subject &lt;#it&gt;.
-           &lt;#it&gt;
-             trip:attachmentList (
-             &lt;http://www.zenith.org/&gt;
-             &lt;https://webfoundation.org/&gt;
-             &lt;http://dbp.de/&gt;
-             &lt;http://cbc.ca/&gt;
-             &lt;http://bbc.co.uk/&gt;
-             &lt;http://aclu.org/&gt;
-             &lt;http://acm.org/&gt;
-              ) .
-
-</pre>
-          </td>
-
-          <td class='output'></td>
-        </tr>
-
-        <tr>
-          <td class='input'>
-            <pre>
-         # Remove all but one
-         &lt;#tabs&gt; a ui:Tabs;
-           ui:predicate trip:attachmentList;
-           ui:orientation 3; ui:ordered true;
-           ui:subject &lt;#it&gt;.
-         &lt;#it&gt;
-           trip:attachmentList ( &lt;https://webfoundation.org/&gt; ) .
 
-</pre>
-          </td>
+<h1>The Tabs module</h1>
+
+<p>
+    On this page you'll find some examples from the Tabs module.
+</p>
+
+<article id="tabWidget-with-collection">
+    <h2>
+        <a href="#tabWidget-with-collection">
+            <code>tabWidget</code>: Populating content using a RDF collection
+        </a>
+    </h2>
+    <p>
+        In this first example we'll see how you can generate a tabs widget using triples that's formatted as a
+        collection
+        in RDF. See <a href="http://127.0.0.1:8080/Documentation/api/modules/_tabs_.html#tabwidget">API
```

Comment:
Link to https://solid.github.io/solid-ui/Documentation/api/modules/_tabs_.html#tabwidget instead